Dockerfile-alpine.template

```diff
@@ -52,11 +52,19 @@ RUN set -ex \
 		tk-dev \
 		xz-dev \
 		zlib-dev \
+                libffi-dev \
+                expat-dev \
 # add build deps before removing fetch deps in case there's overlap
 	&& apk del .fetch-deps \
 	\
 	&& cd /usr/src/python \
+        && rm -r Modules/expat \
+		Modules/zlib \
+		Modules/_ctypes/darwin* \
+		Modules/_ctypes/libffi* \
 	&& ./configure \
+		--with-system-ffi \
+                --with-system-expat \
```

Comment:
How will these changes affect the other versions of alpine python (3.3, 3.4, and 3.5)?